sechub-solution/docker/run.sh

```diff
@@ -1,119 +1,175 @@
+
+  echo "Starting up SecHub server"
+  java $java_debug_options $database_options $storage_options \
+    -Dfile.encoding=UTF-8 \
+    -Dspring.profiles.active="$profiles" \
+    -Dsechub.targettype.detection.intranet.hostname.endswith=intranet.example.org \
+    -Dsechub.config.trigger.nextjob.initialdelay=0 \
+    -Dsechub.initialadmin.userid="$ADMIN_USERID" \
+    -Dsechub.initialadmin.email=sechubadm@example.org \
+    -Dsechub.initialadmin.apitoken="$ADMIN_APITOKEN" \
+    -Dspring.servlet.multipart.max-file-size="$SECHUB_MAX_FILE_UPLOAD_SIZE" \
+    -Dspring.servlet.multipart.max-request-size="$SECHUB_MAX_FILE_UPLOAD_SIZE" \
+    -Dsechub.upload.binaries.maximum.bytes="$SECHUB_UPLOAD_BINARIES_MAXIMUM_BYTES" \
+    -Dsechub.adapter.netsparker.userid=abc \
+    -Dsechub.adapter.netsparker.apitoken=xyz \
+    -Dsechub.adapter.netsparker.baseurl=https://example.org \
+    -Dsechub.adapter.netsparker.defaultpolicyid=example \
+    -Dsechub.adapter.netsparker.licenseid=example \
+    -Dsechub.adapter.nessus.defaultpolicyid=example \
+    -Dsechub.notification.email.administrators=example@example.org \
+    -Dsechub.notification.email.from=example@example.org \
+    -Dsechub.notification.smtp.hostname=example.org \
+    -Dserver.port=8443 \
+    -Dserver.address=0.0.0.0 \
+    -jar /sechub/sechub-server*.jar
 }
 
-debug () {
-    wait_loop
+startup_server() {
+  check_variable "$SPRING_PROFILES_ACTIVE" "SPRING_PROFILES_ACTIVE"
+
+  # Initial job scheduling settings
+  init_scheduler_settings
+
+  if [ -n "$SPRING_DATASOURCE_URL" ] ; then
+    check_variable "$SPRING_DATASOURCE_USERNAME" "SPRING_DATASOURCE_USERNAME"
+    check_variable "$SPRING_DATASOURCE_PASSWORD" "SPRING_DATASOURCE_PASSWORD"
+    cat - <<EOF
+Database:
+- connection string: $SPRING_DATASOURCE_URL
+- database user: $SPRING_DATASOURCE_USERNAME
+EOF
+  fi
+
+  if [ -n "${SECHUB_SERVER_SSL_KEYSTORE_ALIAS}" -a "${SECHUB_SERVER_SSL_KEYSTORE_ALIAS}" != "undefined" ] ; then
+    # Create symlink to .p12 keystore file
+    ln -s /sechub/secrets/secret-ssl/sechub_server_ssl_keystore_file ${SECHUB_SERVER_SSL_KEYSTORE_LOCATION}
+    cat - <<EOF
+SSL server certificate:
+- alias: $SECHUB_SERVER_SSL_KEYSTORE_ALIAS
+- location: $SECHUB_SERVER_SSL_KEYSTORE_LOCATION
+EOF
+  fi
+
+  cat - <<EOF
+SecHub server settings:
+- Activated Spring profiles: $SPRING_PROFILES_ACTIVE
+- base url: $SECHUB_SERVER_BASEURL
+- administration url: $SECHUB_SERVER_ADMINISTRATION_BASEURL
+- Upload source code maximum size: $SPRING_SERVLET_MULTIPART_MAX_FILE_SIZE
+- Upload binaries maximum bytes: $SECHUB_UPLOAD_BINARIES_MAXIMUM_BYTES
+- Job scheduling is activated every ${SECHUB_CONFIG_TRIGGER_NEXTJOB_DELAY}ms
+- Job scheduling initial delay: ${SECHUB_CONFIG_TRIGGER_NEXTJOB_INITIALDELAY}ms
+
+Starting up SecHub server
+EOF
+  java $java_debug_options $storage_options \
+    -Dfile.encoding=UTF-8 -Djava.security.egd=file:/dev/urandom -XX:InitialRAMPercentage=50 -XX:MaxRAMPercentage=80 -XshowSettings:vm \
```

Comment:
"Typically, if we're running Java 8 or later and are running on Linux, then our JVM will use file:/dev/urandom by default."(source: https://www.baeldung.com/java-security-egd)

```suggestion
    -Dfile.encoding=UTF-8 -XX:InitialRAMPercentage=50 -XX:MaxRAMPercentage=80 -XshowSettings:vm \
```

Reply:
ok. removed `-Djava.security.egd`